custom/icds_reports/migrations/sql_templates/create_tables_and_views.sql

```diff
@@ -55,6 +55,9 @@ CREATE OR REPLACE VIEW awc_location_months AS
 	months.month_name AS month_display
   FROM awc_location awc_location
   CROSS JOIN "icds_months" months;
+CREATE INDEX "awc_location_months_indx1" ON "awc_location_months" (aggregation_level, month);
+CREATE INDEX "awc_location_months_indx2" ON "awc_location_months" (aggregation_level, state_site_code, month);
+CREATE INDEX "awc_location_months_indx3" ON "awc_location_months" (aggregation_level, state_id, district_id, block_id, supervisor_id, awc_id, month);
```

Comment:
It's been a while since I took a deep look at these scripts, but awc_id isn't actually unique on this table.

Aggregation level is an integer that says if the row of data is an aggregate of data for the state, district, block, supervisor or awc.

so the table  could be

| agglevel | state | district | block | sup | awc | month | # forms |
| --- | -- | -- | -- | -- | -- | -- | -- | 
| 1 | state1 | null | null | null | null | july | # form for the state |
| 5 | state1 | district1 | block1 | sup1 | awc1 | july | # form for the awc |